tests/kuttl/tests/openstackbaremetalset_scale/03-assert.yaml

```diff
@@ -39,17 +40,19 @@ metadata:
   name: compute-provisionserver
   namespace: openstack
 spec:
-  baseImageUrl: http://192.168.111.1/images/rhel-guest-image-8.4-1168.x86_64.qcow2
-  downloaderImageUrl: quay.io/openstack-k8s-operators/rhel-downloader:0.0.1
-  apacheImageUrl: registry.redhat.io/rhel8/httpd-24:latest
-  provisioningAgentImageUrl: quay.io/openstack-k8s-operators/provision-ip-discovery-agent:0.0.1
+  apacheImageUrl: registry.redhat.io/rhel8/httpd-24@sha256:162cc35b2edb9054d29293ed3f0f98f195cf16647e89c3b8c7a4e911606ef4fd                                              
+  baseImageUrl: http://192.168.111.1/images/rhel-guest-image-8.4-1168.x86_64.qcow2                                                                                       
+  downloaderImageUrl: quay.io/openstack-k8s-operators/rhel-downloader@sha256:1a1caa9c2841f6aa66e385c3a21b9d084fd039c02b85e0a78722ededbed0bc22                            
   port: 6190
+  provisioningAgentImageUrl: quay.io/openstack-k8s-operators/provision-ip-discovery-agent@sha256:22db35fb3fb4bb2d67a29f6db46ca8cbd8a903cc4f3437cfaec11f2b6b6d3b6b 
 status:
   conditions:
   - message: ProvisionServer compute-provisionserver has been provisioned
     reason: ProvisionServer compute-provisionserver has been provisioned
     status: "True"
     type: Provisioned
+  localImageUrl: http://172.22.0.220:6190/images/rhel-guest-image-8.4-1168.x86_64.qcow2/compressed-rhel-guest-image-8.4-1168.x86_64.qcow2                                
+  provisionIp: 172.22.0.220
```

Comment:
Is the `172.22.0.220` IP here deterministic?  I just wonder if Metal3 dnsmasq might assign something different via DHCP.

Reply:
probably not, we could just remove checking those 2